app/views/players/registration.html.slim

```diff
@@ -19,6 +19,11 @@
               = button_tag type: :submit, class: 'btn btn-primary' do
                 => fa_icon 'check'
                 | Submit
+    - if @tournament.nrdb_deck_registration?
+      .col-md-6
+        .alert.alert-danger.dontprint role=alert
+          => fa_icon 'exclamation-triangle', class: 'mr-2'
+          | Deck legality is not checked. Please check decks manually.
```

Comment:
how about "not yet checked" instead?

Reply:
same question for the other file.

Reply:
I was trying to limit the amount of space this would take up. It flows onto the next line if I add 'yet'. Do you think that's worth it?

![image](https://github.com/Project-NISEI/cobra/assets/6625241/eb94b2f4-95a7-4c52-9101-f6326df92ce0)